Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:16.13.0-alpine as build
+FROM node:20-alpine as build
```

Comment:
Not directly related to this PR, but can we also update the nginx image to its latest version (`1.25.2-alpine`)?

Reply:
Done